connector/connect/common/src/main/protobuf/spark/connect/base.proto

```diff
@@ -125,6 +125,9 @@ message AnalyzePlanRequest {
 
       // Generates a physical plan outline and also node details.
       EXPLAIN_MODE_FORMATTED = 5;
+
+      // Does not generate any plan when used.
+      EXPLAIN_MODE_SILENT = 6;
```

Comment:
nit: Disabled feels more intuitive to me than silent for the naming.
```suggestion
      EXPLAIN_MODE_DISABLED = 6;
```

Reply:
Are you referring to everywhere, or just here? I'm fine with changing it everywhere, but if that's not the suggestion I think we should keep all references to that mode the same.